api/SpeechGrammar.json

```diff
@@ -0,0 +1,220 @@
+{
+  "api": {
+    "SpeechGrammar": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SpeechGrammar",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "25",
+            "notes": "Speech recognition interfaces are currently prefixed in Chrome, so you'll need to prefix interface names appropriately, e.g. webkitSpeechRecognition; You'll also need to serve your code through a web server for recognition to work. Speech synthesis is fully supported without prefixes.",
+            "prefix": "webkit"
+          },
+          "chrome_android": {
+            "version_added": "64",
+            "notes": "Speech recognition interfaces are currently prefixed in Chrome, so you'll need to prefix interface names appropriately, e.g. webkitSpeechRecognition; You'll also need to serve your code through a web server for recognition to work. Speech synthesis is fully supported without prefixes.",
+            "prefix": "webkit"
+          },
+          "edge": {
+            "version_added": false
+          },
+          "edge_mobile": {
+            "version_added": false
+          },
+          "firefox": {
+            "version_added": "44",
+            "flags": [
+              {
+                "type": "preference",
+                "name": "media.webspeech.recognition.enable",
+                "value_to_set": "true"
+              }
+            ],
+            "notes": "Note that currently only the speech synthesis part is available in Firefox Desktop — the speech recognition part will be available soon, once the required internal permissions are sorted out."
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "27"
+          },
+          "opera_android": {
+            "version_added": false
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "SpeechGrammar": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/SpeechGrammar",
```

Comment:
Should be "https://developer.mozilla.org/docs/Web/API/SpeechGrammar/SpeechGrammar"
And a description should be added:
"description": "<code>SpeechGrammar</code> constructor",